.circleci/config.yml

```diff
@@ -152,7 +152,7 @@ jobs:
       TERM: xterm
       CC: /usr/bin/clang-7
       CXX: /usr/bin/clang++-7
-      CMAKE_OPTIONS: -DLLL=ON
+      CMAKE_OPTIONS: -DSANITIZE=address -DCMAKE_BUILD_TYPE=Debug
```

Comment:
Relaying the argument of `-DSANITIZE` to 
https://github.com/ethereum/solidity/blob/develop/cmake/EthCompilerSettings.cmake#L141 may be a problem if we want to add a UBSan CI later on.

```
if (SANITIZE)
	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-omit-frame-pointer -fsanitize=${SANITIZE}")
endif()
```

For example, if `SANITIZER=address` we may want to add additional flags for debugging. And if `SANITIZER=undefined` we may want to use several undefined flags such as those listed here: https://github.com/google/oss-fuzz/blob/master/infra/base-images/base-builder/Dockerfile#L22

Reply:
Yes this is a known issue. Can you open an issue for it? (It is not related to this PR, but our CMake options).

Reply:
okay, filed here: #5934